install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+API_FOLDER=~/.ArduinoCore-API
+if [ ! -d "$API_FOLDER" ] ; then
+	echo "Cloning ArduinoCore API in" $API_FOLDER
+	git clone git@github.com:arduino/ArduinoCore-API ~/.ArduinoCore-API
+else
+	echo "API Folder already exists, skipping clone..."
+fi
+
+echo "Commenting out WCharacter.h because it fails to build properly"
+sed '/WCharacter.h/ s/./\/\/ &/' ~/.ArduinoCore-API/api/ArduinoAPI.h > ~/.ArduinoCore-API/api/tmpArduinoAPI.h
+mv ~/.ArduinoCore-API/api/tmpArduinoAPI.h ~/.ArduinoCore-API/api/ArduinoAPI.h
+
+echo "\n\nLinking..."
```

Comment:
```suggestion
echo -e "\n\nLinking..."
```

`echo` needs `-e` to process `\n` as a newline character

```
$ echo "\nNewline Test\n"
\nNewline Test\n
$ echo -e "\nNewline Test\n"                     

Newline Test

```